code/modules/mob/living/simple_animal/hostile/infestation/lurker.dm

```diff
@@ -0,0 +1,82 @@
+// High ranged damage, Medium health, Low mobility.
+// Recommended strategy is to rush it, as its melee damage is subpar and has no armor penetration.
+/mob/living/simple_animal/hostile/infestation/lurker
+	name = "lurker"
+	desc = "A spider-like creature with multiple spikes protruding from its \"legs\"."
+	icon = 'icons/mob/simple_animal/abominable_infestation/48x48.dmi'
+	// !!!! I am not a very good spriter, please update it later !!!!
```

Comment:
```suggestion
	// TODO: improve these sprites
```

or better yet make an issue (if this is merged)

Reply:
Lurker is not available normally so this is fine for now.